### PR TITLE
doc: known issues: Add KRKNWK-19263 related to Zigbee FOTA bug on nRF53

### PR DESCRIPTION
**Pull request by AdamZelikNS**
Added KRKNWK-19263 as a known issue.

### CODE REVIEW

**Comment by AdamZelikNS** [comment]
Fix mentioned as the "workaround" provided here https://github.com/nrfconnect/sdk-nrf/pull/17796. Waiting for merge and then I will update the hash

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -1425,6 +1425,13 @@ KRKNWK-7836: Coordinator asserting when flooded with ZDO commands
 KRKNWK-6073: Potential delay during FOTA
   There might be a noticeable delay (~220 ms) between calling the ZBOSS API and on-the-air activity.
 
+.. rst-class:: v2-7-0
+
+KRKNWK-19263: FOTA DFU on nRF5340 fails due to invalid update image
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:1430]:
```suggestion
KRKNWK-19263: FOTA DFU on the nRF5340 DK fails due to an invalid update image
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 2]
```diff
@@ -1425,6 +1425,13 @@ KRKNWK-7836: Coordinator asserting when flooded with ZDO commands
...
+  Performing fota dfu on nrf5340 throws an error during an attempt to install the new image.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:1431]:
```suggestion
  Performing FOTA DFU throws an error during an attempt to install the new image.
  
  **Affected platforms:** nRF5340
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 3]
```diff
@@ -1425,6 +1425,13 @@ KRKNWK-7836: Coordinator asserting when flooded with ZDO commands
...
+
+  **Workaround:** Manually cherry-pick and apply commit with fix from main (commit hash: ``to be updated: pull/17796 not yet merged``).
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:1433]:
```suggestion
  **Workaround:** Manually cherry-pick and apply the commit with the fix from ``main`` (commit hash: ``to be updated: pull/17796 not yet merged``).
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 4]
```diff
@@ -1425,6 +1425,13 @@ KRKNWK-7836: Coordinator asserting when flooded with ZDO commands
 KRKNWK-6073: Potential delay during FOTA
   There might be a noticeable delay (~220 ms) between calling the ZBOSS API and on-the-air activity.
 
+.. rst-class:: v2-7-0
```
[the remark this answers is not in the document]
Comment (AdamZelikNS) [reply doc/nrf/releases_and_maturity/known_issues.rst]: 👍 done

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 1 above)
Comment (AdamZelikNS) [reply doc/nrf/releases_and_maturity/known_issues.rst]: 👍 fixed

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 2 above)
Comment (AdamZelikNS) [reply doc/nrf/releases_and_maturity/known_issues.rst]: fixed

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 3 above)
Comment (AdamZelikNS) [reply doc/nrf/releases_and_maturity/known_issues.rst]: fixed

**Review by wiba-nordic — APPROVED** [review]

**Review by edmont — APPROVED** [review]

**Pull request merged**